src/OpenTelemetry.Exporter.InMemory/InMemoryExporter.cs

```diff
@@ -35,6 +37,11 @@ public override ExportResult Export(in Batch<T> batch)
                 return ExportResult.Failure;
             }
 
+            if (this.isMetric)
+            {
+                this.exportedItems.Clear();
```

Comment:
Might be worth adding a comment here briefly explaining why the special clear for metrics.

Reply:
Done. Can you please proofread what I wrote? I have a tendency to be verbose. :)

"By design, The MetricApi reuses Metrics (MetricReader.metricsCurrentBatch). This means that exported Metrics will always reflect the latest values. Here, we clear the exported collection to prevent populating this with duplicate instances of the same Metric."